apps/vth-dashboard/src/api/sub-thema/content-types/sub-thema/schema.json

```diff
@@ -13,6 +14,14 @@
   "attributes": {
     "Titel": {
       "type": "string"
+    },
+    "Inhoud": {
+      "type": "richtext"
+    },
+    "slug": {
```

Comment:
I think you need to use this plugin https://market.strapi.io/plugins/strapi-plugin-slugify to handle the slug.
We have already used it in the PDC dashboard.
`vth-dashboard/config/plugins.ts`

```ts
  slugify: {
    enabled: true,
    config: {
      shouldUpdateSlug: true,
      contentTypes: {
        thema: {
          field: 'slug',
          references: 'title',
        },
        'thema-content': {
          field: 'slug',
          references: 'title',
        },
        'sub-thema': {
          field: 'slug',
          references: 'title',
        },
      },
    },
  },
  
```